src/EnergyPlus/HeatPumpWaterToWaterHEATING.cc

```diff
@@ -753,7 +722,7 @@ void GshpPeHeatingSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad)
         while (true) {
             CompSuctionTemp = 0.5 * (T110 + T111);
 
-            CompSuctionEnth = GetSupHeatEnthalpyRefrig(
+            CompSuctionEnth = FluidProperties::GetSupHeatEnthalpyRefrig(
                 state, GSHPRefrigerant, CompSuctionTemp, SuctionPr, state.dataHPWaterToWaterHtg->GSHPRefrigIndex, RoutineNameCompSuctionTemp);
             if (std::abs(CompSuctionEnth - SuperHeatEnth) / SuperHeatEnth < 0.0001) {
                 goto LOOP_exit;
```

Comment:
just use `break`. `goto`'s are okay (I guess) for breaking out of nested loops in one shot, but this isn't that setup.